### PR TITLE
Lint

### PR DESCRIPTION
**Pull request by e-dard**
Supports #4098. Lints the following packages:

- `monitor`;
- `services/continuous_querier` :exclamation:;
- `services/graphite` :exclamation:;
- `services/hh` :exclamation:;
- `services/httpd`;
- `services/graphite`; and
- `services/opentsdb`.

`services` also contains the `copier/internal` package, which is generated, so that's obvious not been linted.

Packages with :exclamation: have outstanding lint issues. Specifically:
```
continuous_querier/config.go:1:1: don't use an underscore in package name
continuous_querier/service.go:1:1: don't use an underscore in package name
continuous_querier/service_test.go:1:1: don't use an underscore in package name
graphite/parser.go:192:78: exported func NewTemplate returns unexported type *graphite.template, which can be annoying to use
hh/limiter.go:15:34: exported func NewRateLimiter returns unexported type *hh.limiter, which can be annoying to use
```

Because the tests in these packages use the exported test package idiom, the `NewX` functions cannot be unexported as it stands. I'm not sure whether they're supposed to be part of the external API or not regardless. 

In terms of the `continuous_querier` package I can rename this if it suits—depends on if it's name is underscored for a reason.

If anyone wants the remaining issues resolved: chime up, otherwise these packages are done.

### CODE REVIEW

File: services/graphite/parser.go [hunk 1]
```diff
@@ -157,8 +159,8 @@ func (p *Parser) Parse(line string) (models.Point, error) {
 	return models.NewPoint(measurement, tags, fieldValues, timestamp)
 }
 
-// Apply extracts the template fields form the given line and returns the
-// measurement name and tags
+// ApplyTemplate extracts the template fields form the given line and
```
Comment (otoolep) [inline services/graphite/parser.go]: form -> from

File: services/graphite/service.go [hunk 2]
```diff
@@ -377,6 +379,7 @@ func (s *Service) processBatches(batcher *tsdb.PointBatcher) {
 	}
 }
 
+// Diagnostics returns diagnositics of the graphite service.
```
Comment (otoolep) [inline services/graphite/service.go]: diagnositics -> diagnostics

File: services/hh/service.go [hunk 3]
```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/influxdb/influxdb/monitor"
 )
 
+// ErrHintedHandoffDisabled is returned when attempted to write to a
```
Comment (otoolep) [inline services/hh/service.go]: -> when attempting to write to use the a disabled hinted handoff service.

File: services/hh/service.go [hunk 4]
```diff
@@ -72,6 +75,7 @@ func NewService(c Config, w shardWriter, m metaStore) *Service {
 	}
 }
 
+// Open attempts to start the hinted handoff service.
```
Comment (otoolep) [inline services/hh/service.go]: There is no try. Do or do not. :-)

"Open opens the hinted handoff service"

File: services/hh/service.go [hunk 5]
```diff
@@ -119,6 +123,7 @@ func (s *Service) Open() error {
 	return nil
 }
 
+// Close attempts to gracefully shutdown the hinted handoff service.
```
Comment (otoolep) [inline services/hh/service.go]: Ditto.

File: services/httpd/handler.go [hunk 6]
```diff
@@ -29,8 +29,11 @@ import (
 )
 
 const (
-	// With raw data queries, mappers will read up to this amount before sending results back to the engine.
-	// This is the default size in the number of values returned in a raw query. Could be many more bytes depending on fields returned.
+	// DefaultChunkSize specified the amount of data mappers will read
```
Comment (otoolep) [inline services/httpd/handler.go]: specified -> specifies

File: services/httpd/handler.go [hunk 7]
```diff
@@ -617,6 +621,7 @@ func MarshalJSON(v interface{}, pretty bool) []byte {
 	return b
 }
 
+// Point represents an influx point.
```
Comment (otoolep) [inline services/httpd/handler.go]: influx -> InfluxDB

File: services/httpd/handler.go [hunk 8]
```diff
@@ -624,6 +629,8 @@ type Point struct {
 	Fields map[string]interface{} `json:"fields"`
 }
 
+// Batch is a collection of points belonging to a database, with a
```
Comment (otoolep) [inline services/httpd/handler.go]: belonging to -> associated with

**Comment by otoolep** [comment]
Thanks @e-dard -- some minor feedback. I can merge once addressed.

**Comment by e-dard** [comment]
Thanks @otoolep. Sorry about the typos. 

"I'll just lint this package on a Sunday evening, it'll only take five minutes"... Thirty minutes later.. Arggh!

**Comment by e-dard** [comment]
Sorry for the late catchup on this; my daughter decided to arrive 3 weeks early :baby: Should be all good now @otoolep ?

**Comment by otoolep** [comment]
Looks good to me @e-dard -- thanks. +1

@corylanou ? Please merge if you're OK with this.

**Pull request merged**